lib/output-files.js

```diff
@@ -24,8 +25,19 @@ class OutputFiles {
 
     var str = ``
 
-    // Get the last element in the path name
-    var truncatedPath = pathLib.basename(path)
```

Comment:
use `let` instead of `var`, if you don't mind 👍`const` and `let` are scoped to the block they're declared in, making it harder to leak information.

Reply:
Fixed!